src/javascript/binary/base/page.js

```diff
@@ -162,12 +162,8 @@ Page.prototype = {
         }
     },
     general_authentication_message: function() {
-        const div = $('<div/>', { text: localize('Please send us the following documents in order to verify your identity and authenticate your account:') });
```

Comment:
hold on, why are we removing the authentication message? It will slow things down if clients have to contact CS for the instructions